react_ws_src/src/views/ttt/SetGameType.js

```diff
@@ -17,6 +17,10 @@ export default class SetGameType extends Component {
 				<h1>Choose game type</h1>
 
 				<button type='submit' onClick={this.selTypeLive.bind(this)} className='button long'><span>Live against another player <span className='fa fa-caret-right'></span></span></button>
+				&nbsp;&nbsp;&nbsp;&nbsp;
+				<button type='submit' onClick={this.selTypeHost.bind(this)} className='button long'><span>Host game<span className='fa fa-caret-right'></span></span></button>
+				&nbsp;&nbsp;&nbsp;&nbsp;
+				<button type='submit' onClick={this.selTypeJoin.bind(this)} className='button long'><span>Join game<span className='fa fa-caret-right'></span></span></button>
```

Comment:
adding new feature to host a game

<img width="1345" alt="Screen Shot 2022-06-30 at 9 38 22 pm" src="https://user-images.githubusercontent.com/48106464/176668296-ab2d0aec-ac04-4bef-a4cd-0a9615726a98.png">